manuscript.Rmd

```diff
@@ -1165,6 +1165,29 @@ This analysis plan may contain data-dependent decisions to increase flexibility
 
 ## Limitations
 
+We have to distinguish two kinds of limitations.
+First, some limitations are by design and are unlikely to be changed soon.
+Our workflow does inherit these from the software it relies on and the fundamental design principles these share with the workflow and `repro`.
+Second, there are other limitations that we hope development effort of the open-source community and, concerning `repro`, us will ease.
+
+As a principle, this workflow is incompatible with steps that cannot be automated.
+This principle is at odds with some ingrained habits of researchers to mix and match manual and automated steps in data analysis.
+To allow for automation, many researchers will have to search for alternative software.  
+The automation friendly software we present here has several technical but critical limitations.
+`Git` can track any files (files bigger than 1GB only with extensions) but can only compare text files (with endings like, `.txt`, `.csv`, `.R`,  or `.Rmd`), not binary files (with endings like `.docx`, `.exe` or `.zip`).
+With `R Markdown`, tables and graphics are dynamically generated from code but cannot be crafted by hand.
+`Make` can orchestrate any programmable software but fails to automate software that will only obey a user clicking at the right buttons.
+`Docker` can ship software that runs on Linux and can be automatically installed, but that requirement precludes many commercial or closed source software.
+
+This move away from software that has served researchers well for decades is understandably difficult and presents us with a conundrum.
+On the one hand, we firmly believe that automated reproducibility makes research more productive and collaboration easier.
+But, on the other hand, we expect researchers to invest considerable time in learning new tools and which they then must "force" on their collaborators.
```

Comment:
I would remove or rephrase "force on their collaborators". I understand the sentiment you are trying to convey, but 1) the phrase is too colloquial for a scientific paper and 2) the issue of collaboration is complex and important enough to merit its own paragraph (or even paper), not to be mentioned as an afterthought.

Reply:
I see. I still want to put these fears plainly on the table because our readers will have them. Do you have an alternative phrasing for this in mind?

Anyway, we removed the longer collaboration section to tighten the paper.

I admit I am biased, but except for you both, I force no one to use these tools but get along by integrating these into a mix and match workflow. Often my collaborator even do not realize that I use version control + docker + make till they have problems reproducing something.